src/main.rs

```diff
@@ -33,6 +40,13 @@ fn main() -> Result<(), Box<dyn Error>> {
 
             process::exit(0);
         }
+        ("compare", Some(files)) => {
+            if let Ok(warnings) = dotenv_linter::compare(&files, &current_dir) {
+                if warnings.is_empty() {
+                    process::exit(0);
+                }
+            }
```

Comment:
```suggestion
            let warnings = dotenv_linter::compare(&files, &current_dir)?;
            if warnings.is_empty() {
                process::exit(0);
            }
```
I think it will be a little cleaner this way 🙂